src/stream_processor/parser/sql.l

```diff
@@ -85,11 +85,11 @@ UNIX_TIMESTAMP          return UNIX_TIMESTAMP;
 RECORD_TAG              return RECORD_TAG;
 RECORD_TIME             return RECORD_TIME;
 
-"true"                  { yylval->boolean = true;  return BOOLEAN; };
-"false"                 { yylval->boolean = false;  return BOOLEAN; };
+"true"                  { yylval->boolean = true;  return BOOLTYPE; };
+"false"                 { yylval->boolean = false;  return BOOLTYPE; };
```

Comment:
Is there a particular Windows head that can be avoided for this?

The [documentation](https://docs.microsoft.com/en-us/windows/desktop/winprog/windows-data-types) indicates that BOOLEAN is defined in `WinNT.h`  By the look of the [mingw winnt.h](https://github.com/Alexpux/mingw-w64/blob/master/mingw-w64-tools/widl/include/winnt.h) we'd need this header to talk to Win32 APIs, but possibly here we could `#define _WINNT_` to skip all the namespace pollution.

Reply:
We use VC++ not MinGW.
I cannot prevent including Windows.h with your suggestion.

Reply:
What's the include guard used by VC++ with the Windows SDK?  Likely it's the same as what mingw uses.

Reply:
Windows.h pollution is originated from mk_core.h which includes mk_sleep.h.
We shouldn't use just a workaround patch.

Reply:
I've investigated this pollution issue, but this pollution is very tightened to monkey Windows support.
Do you have any good idea to solve this?